### ISSUE TITLE
size_constraint_weight parameter

### ISSUE THREAD

**Issue by manuel-88** [ferdyandannes/struct2depth_train#2]
Hi, in your "use" text file where the parameters are listed you used, there you set the _size_constraint_weight_  to zero. According to the parameter description, it defines the weight of the objects and is only used when motion handling is enabled. If it is set to zero isn't this parameter then giving zero weight to every object and therefore the segmentation masks are not included in the training?

**Comment by ferdyandannes** [comment]
> Hi, in your "use" text file where the parameters are listed you used, there you set the _size_constraint_weight_ to zero. According to the parameter description, it defines the weight of the objects and is only used when motion handling is enabled. If it is set to zero isn't this parameter then giving zero weight to every object and therefore the segmentation masks are not included in the training?

Yes, that is one of my problems, if I not set to zero, the training gives me an error.
I still figured it out how to solve it, once I successfully train it with size_constraint_weight, but with the mask image only consist of one object only, if in the mask image consist of 2 or more object it gives me an error.
